src/components/application_manager/src/commands/mobile/perform_audio_pass_thru_request.cc

```diff
@@ -426,7 +427,27 @@ PerformAudioPassThruRequest::PrepareAudioPassThruResultCodeForResponse(
       (tts_result != hmi_apis::Common_Result::INVALID_ENUM)) {
     common_result = hmi_apis::Common_Result::WARNINGS;
     out_result = true;
-  } else if (ui_result == hmi_apis::Common_Result::INVALID_ENUM) {
+  }
+
+  else if ((ui_result == hmi_apis::Common_Result::WARNINGS ||
+            tts_result == hmi_apis::Common_Result::WARNINGS) &&
+           Compare<hmi_apis::Common_Result::eType, EQ, ONE>(
```

Comment:
@VVeremjova
The case when ui_result is WARNINGS is covered by the last else statement (line 452), so here it must be excluded. I think you must check if ui_result.is_ok and tts_result == WARNINGS.